pages/tidal tools/analyze_database.md

```diff
@@ -151,73 +188,61 @@ host    all         tidal   0.0.0.0/0   md5
 
 To apply the configuration changes to the running PostgreSQL server you will need to run [`pg_ctl restart`](https://www.postgresql.org/docs/current/app-pg-ctl.html).
 
-## Perform the analysis
-
-With your user and password, you can define all these values in a YAML configuration file.
 
-The simplest way is to use `tidal analyze db init` and answer the questions. Or you can create the file manually:
 
+## Perform the analysis
 
-databases.yaml:
+When analyzing a database (or multiple), You have two options. 
 
-```yaml
-databases:
-  - id: 111
-    engine: Oracle
-    host: 'my-db-host.com'
-    port: 1521
-    db_name: 'orcl'
-    user: 'tidal'
-    password: 'yoursecurepassword1234!'
-    name: 'My-Test-DB'
+1. Perform the database analysis and upload the result immediately to your workspace. This is Tidal Tools default behaviour. To do so, all you need to do is run the following command.
+```bash
+tidal analyze db databases.yaml
 ```
 
-{% include note.html content="It is best to use quotations, either double or single, around the values in the configuration file. To avoid special characters, : `{ } [ ] , & * # ? | - < > = ! % @ \ \n` from being interpreted. Single quotes are safest, if the value has a single quote within it, you can include it by using a two single quotations, ie. `'my''string'` - will become `my'string`." %}
+2. Perform the database analysis and upload the results at a later time. The following section will explain how to run database analysis in **offline mode**
 
-{% include tip.html content="Are you analyzing an Oracle Stanard Edition (SE)
-database or using a CDB database? Check out the [advanced configuration below](#advanced-configuration)." %}
+### Running offline
```

Comment:
I think it's starting to get a bit confusing having a dedicated 'tidal offline' page and also including offline instructions in place. [This section](https://guides.tidalmg.com/tidal-offline.html#database-analysis) is also about analyzing databases offline, and will now be out of date. Maybe we can decide whether to keep all offline instructions in a single place, or remove that page and include the instructions in their respective guides?

Reply:
Alternatively, maybe we can link from the tidal offline page to this new section.

Reply:
Great thinking Sam.

For context. On how people generally go through the setup and process.

One of the first steps is 'deployment of tidal'. - This is where the offline guide is intended to be used. It should work to help people with the question, "Ok I need to setup _all_ the Tidal Migrations discovery tools [and any dependencies] in our environment, and for it to work without direct internet access. How can I do that?"

Later they will go through each discovery steps (machine stats, source code analysis, db, etc), sometimes skipping the ones that are not relevant. The actual guides for each discovery/analysis step, ie. database analysis, are of course useful for people when they ask us, "Ok I'm ready to analyze these X databases, how do I do that with tidal migrations?"

Hope that is useful!

Reply:
Thanks for the discussion. We talked (during the sync up) about the pros and cons of having some documentation duplicated. 
With the additional objective of having the guides as building blocks for the PDF Booklet. We understand that it is more beneficial to have some sort of duplication. It will allow the reader to flow through the guides without having to jump back and forth between pages (tabs/links).

I think once we are done with db analysis we can take another pass at the offline guide. see if there are adjustments t to be made.